src/platform/plugins/shared/ui_actions/public/plugin.ts

```diff
@@ -62,11 +61,7 @@ export class UiActionsPlugin
   }
 
   public start(core: CoreStart): UiActionsPublicStart {
-    setAnalytics(core.analytics);
-    setI18n(core.i18n);
-    setNotifications(core.notifications);
-    setTheme(core.theme);
-    setUserProfile(core.userProfile);
+    this.service.start(core);
```

Comment:
This provides the core services to the UIActionsService and the UIActionsExecutionService.